test/tools/OpenCover/OpenCover.psm1

```diff
@@ -486,10 +486,16 @@ function Invoke-OpenCover
             # check to be sure that the module path is present
             # this isn't done earlier because there's no need to change env:PSModulePath unless we're going to really run tests
             $saveModPath = $env:PSModulePath
-            $env:PSModulePath = "${PowerShellExeDirectory}\Modules"
-            if ( ! (test-path $env:PSModulePath) )
+            $env:PSModulePath = "${PowerShellExeDirectory}\Modules;$TestToolsModulesPath"
+
+            $modulePathParts = $env:PSModulePath -split ';'
+
+            foreach($part in $modulePathParts)
```

Comment:
I want this to be fatal error so that we do not have unpredictable results due to a missing path.